app/src/main/java/com/framgia/forder/data/source/local/MyFirebaseInstanceIDService.java

```diff
@@ -10,12 +10,12 @@
 
 public class MyFirebaseInstanceIDService extends FirebaseInstanceIdService {
 
-    private static final String TAG = "MyFirebaseIIDService";
+    private static final String TAG = "MyFirebaseInstanceIDService";
 
     @Override
     public void onTokenRefresh() {
-
-        String refreshedToken = FirebaseInstanceId.getInstance().getToken();
-        Log.d(TAG, refreshedToken);
+        super.onTokenRefresh();
+        String token = FirebaseInstanceId.getInstance().getToken();
+        Log.d(TAG, token);
```

Comment:
ERROR: [LongLogTag] The logging tag can be at most 23 characters, was 27 (MyFirebaseInstanceIDService)
 Log tags are only allowed to be at most 23 tag characters long.